src/app.js

```diff
@@ -0,0 +1,59 @@
+import axios from 'axios';
+
+console.log('Script is running..');
+
+
+const infoCountry = document.getElementById('info-country');
+const search = document.getElementById('country-search');
+search.addEventListener('submit', countrySearch);
+const errorMsg = document.getElementById('error-msg');
+
+function countrySearch(searching) {
+    searching.preventDefault();
+    const input = document.getElementById('input');
+    fetchCountryInfo(input.value);
+    input.value = '';
+}
+
+async function fetchCountryInfo(name) {
+    const URI = `https://restcountries.com/v2/name/${name}`
+    infoCountry.innerHTML = ``;
+    errorMsg.innerHTML = ``;
+
+    try {
+        const response = await axios.get(URI);
+        const country = response.data[0];
```

Comment:
Hier zou je nog spreading kunnen toepassen, zodat je niet overal `country.` voor hoeft te zetten
```suggestion
        const { name, flag, population, region, capital, currencies } = response.data[0];
```